.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ Additionally (see https://symfony.com/releases):
  - Always add tests and ensure they pass.
  - Bug fixes must be submitted against the lowest maintained branch where they apply
    (lowest branches are regularly merged to upper ones so they get the fixes too.)
- - Features and deprecations must be submitted against branch 5.x.
+ - Features and deprecations must be submitted against branch 6.x.
```

Comment:
6.1?

Reply:
I don't know if we need a specific branch here actually, it's already above :man_shrugging:

Reply:
Yes, it's kinda redundant. Maybe we should remove the whole line since it's stated above (except deprecations).

Reply:
Maybe:

```suggestion
 - Features and deprecations must be submitted against the latest branch.
```